hs/src/Reach/Simulator/Core.hs

```diff
@@ -331,7 +372,7 @@ instance Interp DLExpr where
     DLE_ObjectRef _at dlarg str -> do
       ev <- interp dlarg
       case ev of
-        V_Object obj -> return $ obj M.! str
+        V_Object obj -> return $ (fromMaybe (impossible "DLE_ObjectRef") (M.lookup str obj))
```

Comment:
This seems like a pattern begging for a helper function

Reply:
yes